api_tests/src/images.spec.ts

```diff
@@ -0,0 +1,54 @@
+jest.setTimeout(120000);
+
+import { UploadImage, DeleteImage } from "lemmy-js-client";
+import {
+  alpha,
+  setupLogins,
+  unfollowRemotes,
+} from "./shared";
+import fs = require('fs');
+
+beforeAll(async () => {
+  await setupLogins();
+});
```

Comment:
This code is equivalent to the simpler:
```suggestion
beforeAll(setupLogins);
```

Reply:
I copied this from another file but cleaned it up now.